src/componentM/Missions.js

```diff
@@ -1,11 +1,37 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { getMissions } from '../redux/missionsAPI';
+import Mission from './Mission';
 
-const SomethingM = () => (
-  <div>
-    <h3>
-      Missions
-    </h3>
-  </div>
-);
+const Missions = () => {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  const loadMissionsAction = bindActionCreators(getMissions, dispatch);
 
-export default SomethingM;
+  useEffect(() => {
+    loadMissionsAction();
+  }, []);
+
+  return (
+    <div className="missions-list">
+      <table className="missions-table">
+        <tr>
+          <th><h4 className="mission-table-heading">Missions</h4></th>
+          <th className="mission-description"><h4 className="mission-table-heading">Descriptions</h4></th>
+          <th><h4 className="mission-table-heading">Status</h4></th>
+          <th><h4 className="mission-table-heading"> </h4></th>
```

Comment:
This empty  `h4` is doing something? if not could you remove it?

Reply:
Yes, it is for a column containing all the Join/Leave buttons. The header does not have a name.
![image](https://user-images.githubusercontent.com/70488620/136305325-884f17e9-1c49-401c-9305-78d8550c33eb.png)